docs/section19/19-17.md

```diff
@@ -0,0 +1,58 @@
+# Section 19 - Rule 19-17           
+**Schema Version:** 0.0.24  
+**Mandatory Rule:** True    
+**Rule ID:** 19-17           
+**Rule Description:** For baseline system types 9 & 10, the system design supply airflow rates shall be based on the temperature difference between a supply air temperature set point of 105°F and the design space-heating temperature set point, the minimum outdoor airflow rate, or the airflow rate required to comply with applicable codes or accreditation standards, whichever is greater.  
+
+**Rule Assertion:** Options are Pass/UNDETERMINED/NOT_APPLICABLE    
+**Appendix G Section:** G3.1.2.8.2           
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:**  
+1. Applies to baseline HVAC system types 9 & 10.     
+   
+   
+**Function Calls:**  
+1. get_baseline_system_types()    
+2. baseline_system_type_compare()  
+3. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()   
+4. are_all_hvac_sys_fan_objs_autosized() 
+5. get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM()  
+
+## Rule Logic:   
+**Applicability Check 1 - Check if there are any Applicable Baseline System Types in the Project at All**   
+- Create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_9, HVAC_SYS.SYS_10]`  
+- Create dictionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMI)`  
+- Check if B-RMR is modeled with at least one air-side system that is of type 9 or 10, if yes then carry on, if no then NOT_APPLICABLE for the project: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types):` 
+    - Create dictionary of hvac systems and associated zones and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys_x = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`  
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Reset all_design_setpoints_105 boolean to true: `all_design_setpoints_105 = true`  
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `If baseline_system_type_compare(sys_type, HVAC_SYS.SYS_9, false) == TRUE OR baseline_system_type_compare(sys_type, HVAC_SYS.SYS_10, false) == TRUE: `    
+            - Get list of zones served by the hvac system: `zone_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]` 
```

Comment:
System 9 and 10 are single zone systems. Do we expect to get a list or a single zone and a single terminal?